lib/translation_io/client/sync_operation/apply_yaml_source_edits_step.rb

```diff
@@ -49,6 +49,8 @@ def reload_or_reuse_yaml_sources
               yaml_hash      = YAML::load(yaml_content)
               yaml_flat_hash = FlatHash.to_flat_hash(yaml_hash)
 
+              yaml_file_path = yaml_file_path.to_s if yaml_file_path.is_a?(Pathname)
```

Comment:
I could also just get rid of the conditional here, and always coerce the `yaml_file_path` to a String